reddit/uploads/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from reddit.uploads.api.views import UserPostsView, PostCreateView
+
+urlpatterns = [
+    path('user/posts', UserPostsView.as_view(), name='user posts'),
```

Comment:
The first url lists `users` since the view queryset is querying `user` model and not `post` model. 

If you want `posts` the queryset should start from `posts` and not `users`

Reply:
The second url doesn't need `/create` as the POST method on it would imply creation